### PR TITLE
WEBDEV-5018: Restyled collection tile

### PR DESCRIPTION
**Pull request by pezvi**
Updated to match new design

### CODE REVIEW

File: src/tiles/grid/collection-tile.ts [hunk 1]
```diff
@@ -21,18 +21,20 @@ export class CollectionTile extends LitElement {
         }}
```
Comment (jbuckner) [inline src/tiles/grid/collection-tile.ts:21]: Let's remove these `@mouseover` and `@focus` listeners.. I think I put them in trying to debug that Safari issue.

File: src/tiles/grid/collection-tile.ts [hunk 2]
```diff
@@ -43,27 +45,38 @@ export class CollectionTile extends LitElement {
     const cornerRadiusCss = css`var(--collectionTileCornerRadius, 4px)`;
 
     return css`
+      #collection-image-container {
+        /* width: 18rem; */
+        height: 18.5rem;
```
Comment (jbuckner) [inline src/tiles/grid/collection-tile.ts:50]: I think you can remove this height and width with the flex updates suggested below.

File: src/tiles/grid/collection-tile.ts [hunk 3]
```diff
@@ -43,27 +45,38 @@ export class CollectionTile extends LitElement {
...
+        display: flex;
+        justify-content: center;
```
Comment (jbuckner) [inline src/tiles/grid/collection-tile.ts:52]: I fiddled with the layout a bit to allow the tile to grow to its container size (currently it's fixed size) and allows the image to stay centered in the cell if the container changes sizes. Just a few `flex` suggestions:

```suggestion
        justify-content: center;
        align-items: center;
        flex: 1;
```

Allows us to resize the cell and your tile will always fit properly:

Screen Shot 2022-02-03 at 10 33 55

File: src/tiles/grid/collection-tile.ts [hunk 4]
```diff
@@ -43,27 +45,38 @@ export class CollectionTile extends LitElement {
...
         background-size: cover;
       }
 
       #collection-image-title {
         background-color: #666;
-        padding: 1.5rem;
+        border: 1px solid #2c2c2c;
+        padding: 0.5rem;
         border-top-left-radius: ${cornerRadiusCss};
         border-top-right-radius: ${cornerRadiusCss};
```
Comment (jbuckner) [inline src/tiles/grid/collection-tile.ts:71]: Continuing from above:

```suggestion
        border-top-right-radius: ${cornerRadiusCss};
        display: flex;
        flex-direction: column;
        flex: 1;
```

File: src/tiles/grid/collection-tile.ts [hunk 5]
```diff
@@ -73,37 +86,53 @@ export class CollectionTile extends LitElement {
         height: 4rem;
       }
 
+      #container {
+        box-shadow: 1px 1px 2px 0px;
+        border-radius: ${cornerRadiusCss};
```
Comment (jbuckner) [inline src/tiles/grid/collection-tile.ts:91]: Last one:

```suggestion
        border-radius: ${cornerRadiusCss};
        height: 100%;
        display: flex;
        flex-direction: column;
```

**Review by jbuckner — CHANGES_REQUESTED** [review]
The tile is looking really good! A few suggestions, but it's really close!

File: src/tiles/grid/collection-tile.ts (hunk 3 above)
Comment (pezvi) [reply src/tiles/grid/collection-tile.ts:42]: I omitted align-items:center. The collection image is intentionally aligned near the top (to match up with the alignment of other tiles).

**Review by jbuckner — APPROVED** [review]
Nice job! Looks good! 👍

**Pull request merged**